sample/TKAccountSamples.m

```diff
@@ -0,0 +1,176 @@
+//
+//  TKAccountSamples.m
+//  TokenSdkTests
+//
+//  Created by Larry Hosken on 11/3/17.
+//  Copyright © 2017 Token Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "TKTestBase.h"
+
+#import "TokenSdk.h"
+
+// These "tests" are snippets of sample code that get included in
+// our web documentation (plus some test code to make sure the
+// samples keep working).
+
+@interface TKAccountSamples : TKTestBase
+
+@end
+
+@implementation TKAccountSamples
+
+- (void)testGetAccounts {
+    TokenIOSync *tokenIOSync = [[self sdkBuilder] buildSync];
+    TKMemberSync *memberSync = [self createMember:tokenIOSync];
+    [memberSync linkAccounts:[self createBankAuthorization:memberSync]];
+    [memberSync linkAccounts:[self createBankAuthorization:memberSync]];
+    TKMember *member = memberSync.async;
+    NSMutableDictionary *sums = [NSMutableDictionary dictionaryWithCapacity:10];
```

Comment:
Maybe NSMutableDictionary<TKBalance *> is better?

Reply:
oh nice, if I specify type, I can rm ugly casting in other places